Buildings/Experimental/DHC/Loads/Examples/BaseClasses/BuildingTimeSeries.mo

```diff
@@ -120,37 +126,34 @@ model BuildingTimeSeries
     "Heating terminal unit"
     annotation (Placement(transformation(extent={{70,-34},{90,-14}})));
   Buildings.Experimental.DHC.Loads.FlowDistribution disFloHea(
-    redeclare package Medium=Medium,
-    m_flow_nominal=mHeaWat_flow_nominal*facScaHea,
+    redeclare package Medium=MediumW,
+    m_flow_nominal=mHeaWat_flow_nominal,
     have_pum=true,
     dp_nominal=100000,
     nPorts_a1=1,
     nPorts_b1=1) if have_watHea
     "Heating water distribution system"
     annotation (Placement(transformation(extent={{120,-80},{140,-60}})));
   Buildings.Experimental.DHC.Loads.FlowDistribution disFloCoo(
-    redeclare package Medium=Medium,
-    final m_flow_nominal=mChiWat_flow_nominal*facScaCoo,
+    redeclare package Medium=MediumW,
+    m_flow_nominal=mChiWat_flow_nominal,
     typDis=Buildings.Experimental.DHC.Loads.Types.DistributionType.ChilledWater,
     have_pum=true,
     dp_nominal=100000,
     nPorts_b1=1,
     nPorts_a1=1) if have_watCoo
     "Chilled water distribution system"
     annotation (Placement(transformation(extent={{120,-270},{140,-250}})));
-  DHC.Loads.Validation.BaseClasses.FanCoil2PipeCooling terUniCoo(
-    redeclare final package Medium1=Medium,
-    redeclare final package Medium2=Medium2,
+  Buildings.Experimental.DHC.Loads.Validation.BaseClasses.FanCoil2PipeCooling terUniCoo(
+    redeclare final package Medium1=MediumW,
+    redeclare final package Medium2=MediumA,
     final facSca=facScaCoo,
-    final QHea_flow_nominal=QHea_flow_nominal/facScaHea,
-    final QCoo_flow_nominal=QCoo_flow_nominal/facScaCoo,
+    final QCoo_flow_nominal=QCoo_flow_nominal,
```

Comment:
In original implementation, the `QHea_flow_nominal` is specified for each unit, and the `facSca` will scale up the settings to multiple unit.

But in the new change, the `QHea_flow_nominal` is already specified for multiple units and the `facSca` is still there? Again, it seems to related the comment I made previously, what does the scaling factor actually mean?

Reply:
@JayHuLBL See here for the documentation of the scaling factor (if it helps).
https://github.com/lbl-srg/modelica-buildings/blob/master/Buildings/Experimental/DHC/Loads/BaseClasses/PartialTerminalUnit.mo#L562-L592

Reply:
@JayHuLBL do you still want me to elaborate on this item?